brianlong/www.validators.app
# #184205352 - Add score details modal to validator page

#### What's this PR do?
PR provides a modal for validator details page.

#### How should this be manually tested?
- click on the score cycles within validator details page and validators index and verify if the data is valid

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184205352)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
